GitUI/Translation/English.Plugins.xlf

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<xliff xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0" GitExVersion="3.00.rc2">
+<xliff xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0" GitExVersion="3.01.00.0">
   <file datatype="plaintext" original="AppVeyorSettingsUserControl" source-language="en">
     <body>
       <trans-unit id="cbLoadTestResults.Text">
-        <source>Display test results in build status summary for each build result (network intensive!)</source>
+        <source>display tests results in build status summary for each build result (network intensive!)</source>
```

Comment:
I suspect this is unintended since it (I believe) reverts a recent correction.
Writing from a phone, can't check, sorry.

Reply:
.xlf changes are not formally required for the correction as the name was changed too
I ran the translation app to make sure the current status is correct too.

Reverting this line though. This should mean that the correction is incomplete
Other changes should be OK

Reply:
Ah, you didn't indicate you ran the app.
Typically we should run it before a release as strings may change.